src/security/ZeroSensitiveMemory.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 1996-2021 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_SRC_SECURITY_ZEROSENSITIVEMEMORY_H
+#define SQUID_SRC_SECURITY_ZEROSENSITIVEMEMORY_H
+
+#if HAVE_LIMITS_H
+#include <limits.h>
+#endif
+
+#if HAVE_STRING_H
+#include <string.h>
+#endif
```

Comment:
If possible, please use std::memset() and `<cstring>` instead. Besides removing `#include` guards, using a standard C++ function will reduce the number of declaration variation/problems (e.g., I am not sure that C standard guarantees that C memset is not a macro, and there are [claims](https://stackoverflow.com/questions/18754036/macro-definition-for-memset-function-in-c-on-mac-os-x) that it is, in fact, a macro in some Mac environments).